options/locale/locale_en-US.ini

```diff
@@ -944,8 +944,8 @@ archive.title = This repo is archived. You can view files and clone it, but cann
 archive.issue.nocomment = This repo is archived. You cannot comment on issues.
 archive.pull.nocomment = This repo is archived. You cannot comment on pull requests.
 
-form.reach_limit_of_creation_1 = You have already reached your limit of %d repository.
-form.reach_limit_of_creation_n = You have already reached your limit of %d repositories.
+form.reach_limit_of_creation_1 = The owner has already reached the limit of %d repository.
+form.reach_limit_of_creation_n = The owner has already reached the limit of %d repositories.
```

Comment:
But is that typical usage of this error message?
I think this change will confuse users that actually reached the limit themselves:
> Why is the message suddenly talking about some mysterious unknown owner and not me?
Is that message even directed at me?

The best compromise I can come up with is the following: 
```suggestion
form.reach_limit_of_creation_1 = You or the selected owner has already reached the limit of %d repository.
form.reach_limit_of_creation_n = You or the selected owner has already reached the limit of %d repositories.
```

Any better ideas?

Reply:
It's the owner not you. When you create a repo under your username, the limitation depends on you. When you create a repo under an org, the limitation depends on the org's limitation. When you create a repo under other user(you are an admin?), the limitation depends on that user.

Reply:
"You or the selected owner" is a little ambiguous, it sounds like you can't create repo if either you or the owner has reached the limit.

I think the "owner" word may cause a little confusion, but the form is self explanatory, there's still a "owner" selector when the user is creating a repo for himself.

![image](https://user-images.githubusercontent.com/9418365/201250796-87c81a9c-803d-4c6c-947c-000ae9b0acdf.png)